src/ape/api/projects.py

```diff
@@ -115,6 +115,36 @@ def _create_manifest(
 
         manifest.sources = cls._create_source_dict(source_paths, contracts_path)
         manifest.contract_types = contract_types
+
+        manifest.compilers = []
+
+        i = 0
+        for contract_type in contract_types:
+            suffix = source_paths[i].suffix
+            if cls.compiler_manager.registered_compilers[suffix].name:
+                if suffix == ".sol":
+                    compiler_version = [
+                        x
+                        for x in cls.compiler_manager.registered_compilers[suffix].get_version_map(
```

Comment:
It might scale better if we make `get_version_map` a `@raise_note_implemented` method in the `CompilerAPI` and then first attempt to call it and if it raises, then resort to `get_versions()`

Reply:
Edit: does not apply if you do Evan's approach! So ignore this